schema.sql

```diff
@@ -0,0 +1,16 @@
+/*  Execute this file from the command line by typing:
+ *    psql postgres < schema.sql
+*/
+
+DROP DATABASE IF EXISTS airbnb;
```

Comment:
I'd pick a more specific database name. We ran into proxy troubles on FEC when everybody's db was named 'zagat', and I can see the same thing happening here.

Reply:
That's a good point @JacksonTodd93  and I made the same choice (mistake?) as @rogen-code. I don't think we're going to run the proxy locally this time tho. It was only a problem when we tried to have all our four modules on our local machine.